dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,45 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+# configuration elements
+
+concurrencyPolicy=Concurrency Policy
+concurrencyPolicy.desc=A set of behaviors and constraints to be applied for tasks that are capable of asynchronous execution. For example maximum concurrency and maximum queue size.
+
+# attributes
+
+expedite=Expedite
+expedite.desc=Specifies a core number of tasks to aim to run concurrently by expediting requests to the global thread pool. This provides no guarantee that this many tasks will run concurrently. If the expedite value is updated while in use, the update goes into effect gradually as previous expedited and non-expedited requests complete.
+
```

Comment:
No, the core number is a target minimum, not a maximum. I can remove the word 'core' if it was confusing.  If you want to remove, "This provides no guarantee that this many tasks will run concurrently" you need to replace it with something better to get this point across. It's important to point this out so that users don't have a misunderstanding of what they are getting with this attribute.

Reply:
Sounds good, I will make that change